pkg/mount/mountutil.go

```diff
@@ -168,7 +177,10 @@ func getRootPartition() (string, error) {
 	if err != nil {
 		return "", err
 	}
-	defer file.Close()
+
+	defer func() {
+		file.Close()
+	}()
```

Comment:
@asquare14 What is the advantage of using the above format for `defer` compared to
```golang
defer file.Close()
```

Reply:
file.Close() can fail and you need to check the return code. This is not a problem for files opened for reading, but can be  a big problem for files opened for writing. The automated code checkers aren't very smart about this, so they assume the worst: the file is open for writing and you need to deal with failed Close(). 

NB, the above code still ignores the error condition, but it fakes out the automated code checker. Perhaps a better solution is to either explicitly ignore the error condition in the defer func for files opened for read or add a comment so the code checker ignores that check for this instance.

Reply:
@asquare14 Can you make the necessary changes as per the above comment. Here, the file is opened for reading. So,  we can explicitly ignore the error.

Reply:
what I mean by explicitly ignoring for files that are opened for reading is something like:
```
defer func() {
    _ := file.Close()
}
```